KPI/B.adoc

```diff
@@ -1,110 +1,85 @@
 == B. Good quality title
 
-=== To which WMCP element(s) does it apply?
+=== WMCP element(s)
 
-/gmd:MD_Metadata/gmd:identificationInfo//gmd:citation//gmd:title
+`/gmd:MD_Metadata/gmd:identificationInfo//gmd:citation/gmd:CI_Citation/gmd:title`
 
-=== What is being measured?
-* field is not empty 
-* length is not too short or too long. (e.g. at least more than 3 characters and less than 150)
-* words are following Title case 
-* title is following WMCP guidance 
+=== What is being measured
 
-=== Why should we measure it?
-Title is the first reference to a resource displayed and is helps with initial identification. 
-If it's not available, too short, or too long then it makes it difficult for users understand the resource. 
+The title field shall always be populated and and follow the principles of
+WMCP guidance. The length shall not be too short or too long, contain less
+than 3 acronyms and represented in title case.
```

Comment:
Difficult to implement programmatically the acronyms check , unless you have an identified set of acronyms which may be difficult to build ?   We discussed to check whether all words are from an english dictionary (if that makes the check easier)

Reply:
Thanks for the feedback @martaguti.  I've added a check that detects bulletin headers and applies a -1 score.